src/useChildrenAreWrapped/useChildrenAreWrapped.ts

```diff
@@ -0,0 +1,30 @@
+import { computed, ComputedRef, ref, Ref } from 'vue'
+import { useElementRect } from '@/useElementRect/useElementRect'
+import { getComputedStyleValue } from '@/utilities/getComputedStyleValue'
+import { getNumberFromPx } from '@/utilities/getNumberFromPx'
+
+export function useChildrenAreWrapped(children: HTMLElement[] | Ref<HTMLElement[]>, container: HTMLElement | Ref<HTMLElement | undefined>): ComputedRef<boolean> {
+  const childrenRef = ref(children)
+  const containerRef = ref(container)
+
+  const { width } = useElementRect(containerRef)
```

Comment:
I think you can include padding here without to much effort no?

Reply:
good call again